openshift/buildConfig-template.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: template.openshift.io/v1
```

Comment:
please move the openshift templates to https://github.com/thoth-station/thoth-application/pull/30

Reply:
https://github.com/pacospace/thoth-application/pull/1

Reply:
this should be correct PR https://github.com/pacospace/thoth-application/pull/2